### ISSUE TITLE
Relax the tslint selector-prefix 

### ISSUE THREAD

**Issue by nik-lus** [preboot/angular2-webpack#191]
I use this seed quite often and always have to remove the:

"directive-selector-prefix": [true, **"my"**],
"component-selector-prefix": [true, **"my"**],
"pipe-naming": [true, "camelCase", **"my"**]

from the tslint.json 

This isn't a big deal but since the [style docs](https://angular.io/docs/ts/latest/guide/style-guide.html#!#02-07) have gone away from using the prefix 'my' (or 'app') and now recommend using the initials of the project as a prefix I wonder if having the linter test for 'my' is necessary any more.

**Comment by GaryB432** [comment]
I really like the linter rules and just changed my prefix to "the initials of the project" instead of "my"

**Comment by Foxandxss** [comment]
Yes, I just need to explain it nicely in the readme so other people know about it and changes it to what they want, so you get the lint working but your prefix.
